main.js

```diff
@@ -91,22 +91,38 @@ async function main() {
     },
   };
 
+  let hasWebCam = undefined;
+
+  try {
+    const stream = await navigator.mediaDevices.getUserMedia({ video: true });
```

Comment:
Note: `navigator.mediaDevices` works only when the page is loaded under https
>If the current document isn't loaded securely, navigator.mediaDevices will be undefined, and you cannot use getUserMedia(). See Security for more information on this and other security issues related to using getUserMedia().

See more [here](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia)

Reply:
This is great. Thanks for the contribution. Can you push your branch to a new branch on this repo? That'll make it easier for others to test.

Use the following command:
```
git push url://to/new/repository.git branch-to-move:new-branch-name
```

Where `new/repository.git` is this git repo (you can use this: `git@github.com:Global-Innovation-Exchange/FaceSpace.git` ), the `branch-to-move` is the current name of your branch on your repo, and `new-branch-name` can be the same name if you'd like here.

Reply:
Also once you have the new branch setup feel free to close this PR and open a new one on the new branch, just in case other changes get made to the branch. For future development just clone this repository and make changes here to avoid needing to push the branch again :).

Reply:
Thanks wenbo. I think the checking should be done in the `detector.load()` and return a boolean to indicate that since detector.load is doing the same thing.

Reply:
@codycodes Oh I didn't know if I have write permission to this repo.

Reply:
Just added you to this repo. And you can ask Justice for the invite to join the GIX org.